app/models/User.class.js

```diff
@@ -125,6 +126,14 @@ class User {
   getPermissionsForService (serviceExternalId) {
     return _.get(this.getRoleForService(serviceExternalId), 'permissions', []).map(permission => permission.name)
   }
+
+  /**
+   * @method isInternalUser
+   * @returns {boolean}
+   */
+  isInternalUser () {
```

Comment:
Can just access the `.internalUser` property from outside this class without the need fo a getter method. That's what we usually do